### PR TITLE
Move rockskip git calls to git package

### PR DESCRIPTION
**Pull request by mollylogue**
## Test plan

Ensure unit tests/integration tests pass. There should be no functional changes.

### CODE REVIEW

File: enterprise/internal/rockskip/postgres.go [hunk 1]
```diff
@@ -15,6 +15,13 @@ import (
 
 type CommitId = int
 
+type StatusAD int
+
+const (
```
Comment (mollylogue) [inline enterprise/internal/rockskip/postgres.go:20]: I wasn't sure where to put this... everything else moved out of the git.go file and this was the only place these structs were used 🤷‍♀️

**Comment by mollylogue** [comment]
@chrismwendt I tried to add you as a co-author on this (mostly so you show up in the git blame history) but I don't think I did it correctly 🤔

**Comment by chrismwendt** [comment]
@mollylogue Try this:

```
git rebase -i HEAD~2 -x "git -c user.name='Chris Wendt' -c user.email='chrismwendt@gmail.com' commit --amend --reset-author --no-edit"
```

It rewrites the author and committer info for the last 2 commits.

**Review by chrismwendt — APPROVED** [review]

**Comment by mollylogue** [comment]
> ```
> git rebase -i HEAD~2 -x "git -c user.name='Chris Wendt' -c user.email='chrismwendt@gmail.com' commit --amend --reset-author --no-edit"
> ```

Worked like a charm 🙌  Thx

**Pull request merged**